readme-pt-BR.md

```diff
@@ -2,19 +2,32 @@
 
 <img src="./assets/flash.png" width="1000">
 
-[中文版](./README.zh-CN.md) [Русский](./README.ru-RU.md)
-## Introduction
+[中文版](./README.zh-CN.md) [Русский](./README.ru-RU.md) [Português](./README.pt-BR.md)
 
+## Introdução
+
+Este documento contém uma lista de boas práticas que vão nos ajudar a melhorar a performance das nossas aplicações. O "Checklist de Performance do Angular" cobre diferentes tópicos - de pré renderização no servidor e criação do pacote da nossa aplicação até performance em tempo de execu'ão e otimização do sistema de detecção de mudança feita pelo framework.
```

Comment:
using latest orthographic agreement `detecção` should be `deteção`
otherwise `otimização` should be `optimização`

Reply:
Seems like this is a brazilian portuguese translation, so both forms `detecção` and `deteção` are [accepted](https://portuguesaletra.com/acordo-ortografico/deteccao-ou-detecao-ao/). It is just that `deteção` is not used in brazilian portuguese and `detecção` is not used in european portuguese. It is the [same](https://portuguesaletra.com/acordo-ortografico/optimizar-ou-otimizar-ao/) with `otimizar` and `optimizar`.

Reply:
Yes. Its a Brazilian Portuguese translation. I'll update the pull request title.